modelon/impact/client/client.py

```diff
@@ -258,7 +258,7 @@ def get_workspaces(self):
             for item in resp["data"]["items"]
         ]
 
-    def get_project(self, project_id):
+    def get_project(self, project_id: str, vcs_info: bool = True):
```

Comment:
Should default be `True`?

Reply:
Yes, we agreed on having this True in the group chat right?

Reply:
From me trying the client out - yes, True is preferable/safer default. This is mostly to give expected result for someone not reading the docs before trying out the code.